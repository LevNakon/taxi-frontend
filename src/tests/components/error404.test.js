import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Error404 from '../../components/error404';

describe('Error404 --- test render correctly component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Error404 />);
    });

    it('render correctly component Error404', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

    it('h1 exist in component Error404',()=>{
        expect(wrapper.find('h1')).toExist();
    });
});