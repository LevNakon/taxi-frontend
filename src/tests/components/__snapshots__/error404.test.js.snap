// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 --- test render correctly component render correctly component Error404 1`] = `
<h1>
  Page Not Found
</h1>
`;
